storage/object/api-cli/using-api-call-list.mdx

```diff
@@ -0,0 +1,93 @@
+---
+meta:
+  title: Using the Object Storage API
+  description: This page shows Object Storage API operations.
+content:
+  h1: Using the Scaleway Object Storage API
+  paragraph: This page shows Object Storage API operations.
+dates:
+  validation: 2022-04-01
+  posted: 2018-09-13
+---
+
+In this page we index all available Scaleway Object Storage API requests.
+
+<Message type="note">
+
+When you use Object Storage directly with the API, you must generate an [Authentication Signature v4](/storage/object/api-cli/generate-aws4-auth-signature) beforehand.
+
+</Message>
+
+Status:
+  - Feature available: <Icon name="validate" /> 
+  - Feature coming in 2022: ⌛
+
+## Common Operations
+
+| Feature | Description | Status |
+| :------------ | :---- | : - |
+|  [GET Service](/storage/object/api-cli/common-operations/#get-service) | Returns a list of all bucket | <Icon name="validate" /> |
+
+
+## Bucket Operations
+
+| Feature | Description | Status |
+| :------------ | :---- | : - |
+|  [DELETE bucket](/storage/object/api-cli/bucket-operations/#delete-bucket) |  Deletes bucket | <Icon name="validate" /> |
+|  [DELETE bucket CORS](/storage/object/api-cli/bucket-operations/#delete-bucket-cors) |  Deletes the [CORS](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) configuration of a bucket | <Icon name="validate" /> |
+|  [DELETE bucket lifecycle](/storage/object/api-cli/bucket-operations/#delete-bucket-lifecycle)|  Deletes the [lifecycle configuration](/storage/object/api-cli/lifecycle-rules-api/) of a bucket | <Icon name="validate" /> |
+|  [DELETE bucket tagging](/storage/object/api-cli/bucket-operations/#delete-bucket-tagging) |  Deletes the tag(s) of a bucket | <Icon name="validate" /> |
+|  [DELETE bucket website](/storage/object/api-cli/bucket-operations/#delete-bucket-website) |  This operation deletes the bucket Website configuration of a specified bucket | <Icon name="validate" /> |
```

Comment:
check capitalization of Bucket Website (can't remember if it's all upper or all lower?) here and in a few other places in this doc